libs/community/langchain_google_community/drive.py

```diff
@@ -231,6 +337,10 @@ def _load_sheet_from_id(self, id: str) -> List[Document]:
                 }
                 if self.load_auth:
                     metadata["authorized_identities"] = authorized_identities
+                if self.load_extended_metadata:
```

Comment:
can we merge this and above section maybe?
`metadata["owner"] = self._get_owner_metadata_from_id(id)`